charts/datahub/values.yaml

```diff
@@ -491,6 +491,18 @@ global:
       #  - name: "DATAHUB_DB_NAME"
       #    value: "dh"
 
+      ## Use below for usage of Cloud SQL, make sure you have deployed gcloud-sqlproxy as prerequisite
```

Comment:
`Use below for usage of Google Cloud SQL MySQL instance instead of the self hosted MySQL`